cmd/commands/common.go

```diff
@@ -321,7 +321,7 @@ func ensureGitPAT(ctx context.Context, opts *RuntimeInstallOptions) error {
 	}
 
 	if opts.gitProvider != nil {
-		return opts.gitProvider.VerifyToken(ctx, cfgit.PersonalToken, opts.InsCloneOpts.Auth.Password)
+		return opts.gitProvider.VerifyUserToken(ctx, opts.GitIntegrationRegistrationOpts.Token)
```

Comment:
the `--personal-git-token` value ends up in `opts.GitIntegrationRegistrationOpts.Token`.
before - we checked the runtimeToken again by mistake